### DISCUSSION TITLE
How about a WiThrottle version

### DISCUSSION THREAD

**Discussion by AndrewJPye** [DCC-EX/EX-T3-WiFi#121] [Ideas]
I'm so impressed with using this with my DCC-EX that I miss it when using my other command stations that are not DCC-EX. However, I do interface those to JMRI and WiThrottle. I wish my skills were up to being able to create a version that communicates with any WiThrottle server but I am afraid I wouldn't know where to start. Has anyone had a similar idea?

**Comment by alex-code** [comment]
Thanks, it's nice to hear people enjoy controlling their layouts with it.

I wouldn't say it's impossible to support WiThrottle but it would be a lot of work.
Basic control might be simple but there's no provision for fetching rosters etc from the command station.

It's not something I can add but someone else might have the familiarity with WiThrottle to have go.
